pkg/helpers/kubeconfig/loader.go

```diff
@@ -2,25 +2,17 @@ package kubeconfig
 
 import (
 	"github.com/spf13/cobra"
-	"k8s.io/cli-runtime/pkg/genericclioptions"
 
 	kclientcmd "k8s.io/client-go/tools/clientcmd"
-	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
 func NewPathOptions(cmd *cobra.Command) *kclientcmd.PathOptions {
-	return NewPathOptionsWithConfig(kcmdutil.GetFlagString(cmd, genericclioptions.OpenShiftKubeConfigFlagName))
-}
-
-func NewPathOptionsWithConfig(configPath string) *kclientcmd.PathOptions {
```

Comment:
I'm inclined to drop this file entirely. The only difference between this and `kclientcmd.NewDefaultPathOptions` is `LoadingRules.DoNotResolvePaths` but I'd try it, it shouldn't affect us.

Reply:
From what I see we use it in login, logout, project, request project and projects commands.